ingestion/src/metadata/ingestion/source/pipeline/domopipeline/connection.py

```diff
@@ -41,7 +41,7 @@ def test_connection(domo: Domo) -> None:
     Test connection
     """
     try:
-        domo.streams.list()
+        domo.streams.list(limit=1, offset=0)
```

Comment:
@ulixius9 could you add some info as to what we this is doing? 🙂

Reply:
@TeddyCr we were using `domo.streams.list()` method to test connection but [pydomo](https://pypi.org/project/pydomo/#history) lib recently published a new version where they made  `limit` and `offset` as mandatory arguments to be passed due to which the test connection was failing...

but after a detailed look I found that we are not using `pydomo` lib to extract any information instead we are fetching metadata via Domo APIs hence in my next commit I've changed `get_connection` and `test_connection` to interact with rest client